extensions/php/src/features/validationProvider.ts

```diff
@@ -277,10 +285,12 @@ export default class PHPValidationProvider {
 				// Shell args
 				let executableArgs = args.slice(0);
 
-				// Transform Windows file path to Linux file path
-				let windowsPath = executableArgs.pop();
-				let linuxPath = windowsPath.trim().replace(/^([a-zA-Z]):\\/, '/mnt/$1/').replace(/\\/g, '/');
-				executableArgs.push(linuxPath);
+				// If win32 and bash.exe, transform Windows file path to Linux file path
+				if (this.platform === 'win32' && executable.indexOf('bash.exe') !== -1) {
```

Comment:
We can't simply test bash.exe here since git under Window installs a bash.exe as well. We either need to test for the full path or better see how bash from git mount the drive and do the path magic as well.